.github/workflows/release-artifacts.yml

```diff
@@ -25,15 +25,34 @@ jobs:
   build:
     runs-on: ubuntu-22.04
     steps:
+    - name: Debug github event JSON
```

Comment:
Do we need to keep the debug here?

Reply:
I like to add this/similar step to most workflows that depend on inspecting the `event.json`.  Otherwise there's no way to recover that data on job failure should something unexpected happen.  But I can be flexible if you really think it shouldn't persist.